monkey/tests/unit_tests/infection_monkey/credential_store/test_credential_store.py

```diff
@@ -0,0 +1,100 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from infection_monkey.credential_collectors import Password, SSHKeypair, Username
+from infection_monkey.credential_store import AggregatingCredentialsStore
+from infection_monkey.i_puppet import Credentials
+
+DEFAULT_CREDENTIALS = {
```

Comment:
Can we come up with a more descriptive name? `CREDENTIALS_FROM_CONTROL_CHANNEL`? Something better?